src/main/java/com/archyx/aureliumskills/support/PlaceholderSupport.java

```diff
@@ -261,6 +264,15 @@ else if (identifier.equals(skill.name().toLowerCase(Locale.ENGLISH) + "_roman"))
             }
         }
 
+        // Get Actionbar Status
+        if (identifier.startsWith("actionbar_status")) {
+            if (new ActionBar(AureliumAPI.getPlugin()).getPlayerActionbarEnableStatus(player)) {
+                return "Enable";
```

Comment:
Possibly change the wording to either "Enabled"/"Disabled" or to "true"/"false". Your current strings are verbs while the status should be a noun. What is your use case for these placeholders?

Reply:
ok, thank you, i open new pr

Reply:
i wan to show the actionbar status in the menu